mne/utils/docs.py

```diff
@@ -719,22 +723,25 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 # %%
 # D
 
-docdict['dB_plot_psd'] = """
+_dB = """\
+dB : bool
+    Whether to plot on a decibel-like scale. If ``True``, plots
+    10 × log₁₀(spectral magnitude){}.{}
```

Comment:
Should probably be either 20 log10(amplitude) or 10 log10(power)
```suggestion
    20 × log₁₀(spectral amplitude){}.{}
```

Reply:
```suggestion
    10 × log₁₀(spectral power){}.{}
```
Actually the 2 functions where this is used, there is no `estimate='amplitude'` option so it will always be power.